core/ledger/ledger.go

```diff
@@ -177,6 +177,9 @@ func (ledger *Ledger) CommitTxBatch(id interface{}, transactions []*protos.Trans
 	ledger.blockchain.blockPersistenceStatus(true)
 
 	sendProducerBlockEvent(block)
+	if len(transactionResults) != 0 {
+		ledgerLogger.Debug("There were some erroneous transactions. We need to send a 'TX rejected' message here.")
```

Comment:
@srderson Do you mean something like this? https://github.com/hyperledger/fabric/pull/1754